### PR TITLE
Add specific path for Presets

### LINKED ISSUE #1451: "Filter mode" for google.com but "Dynamic" for map?

### LINKED ISSUE #1778: Feature: Add paths as an option below "ONLY FOR DOMAIN.TLD"
I've found many times that I need to apply custom Dark Reader settings to specific paths on sites (admin vs file management vs certain pages, etc.).  Could you please implement below the current "ONLY FOR DOMAIN.TLD" button an additional button or two that would give the current specific page or allow the user to define a specific parental path?

This would allow us to define specific pages or areas of sites that need special attention or have formatting requirements that are different from other areas of a site (such as I said above admin vs user facing frontend).

### PR DESCRIPTION
**Pull request by Gusted**
- Add a specific path to use in Preset dropdown.
- Resolves #1778
- Resolves #1451

### CODE REVIEW

**Comment by Gusted** [comment]
Can't switch from specific -> `host`

**Comment by alexanderby** [comment]
Please explain what you are trying to do, and start from scratch (because of `git push --force` issue).

**Comment by Gusted** [comment]
Now we have default and a the domain as option in the preset picker. This PR adds also the full url so with `/darkreader/darkreader` when on https://github.com/darkreader/darkreader/

File: src/ui/popup/theme/preset-picker/index.tsx [hunk 1]
```diff
@@ -5,12 +5,29 @@ import {ViewProps} from '../../types';
 
 export default function PresetPicker(props: ViewProps) {
     const host = getURLHost(props.tab.url || '');
-    const custom = props.data.settings.customThemes.find(
-        ({url}) => isURLInList(props.tab.url, url)
```
Comment (Gusted) [inline src/ui/popup/theme/preset-picker/index.tsx:9]: You gotta look here what I've changed

**Comment by alexanderby** [comment]
Do you mean something like this?
Screen Shot 2020-07-10 at 14 42 22

**Comment by Gusted** [comment]
Yes, so on this issue it will show `github.com/darkreader/darkreader/pull/2886`.

**Comment by Gusted** [comment]
Closing, thanks.

**Pull request closed without merging**